source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -33,9 +36,76 @@ using Http::RequestTrailerMap;
 using Http::ResponseHeaderMap;
 using Http::ResponseTrailerMap;
 
-static const std::string ErrorPrefix = "ext_proc_error";
-static const int DefaultImmediateStatus = 200;
-static const std::string FilterName = "envoy.filters.http.ext_proc";
+constexpr absl::string_view ErrorPrefix = "ext_proc_error";
+constexpr int DefaultImmediateStatus = 200;
+ABSL_ATTRIBUTE_UNUSED constexpr absl::string_view FilterName = "envoy.filters.http.ext_proc";
```

Comment:
If it is not used,  i think we can just remove it in this PR

Reply:
removed